packages/devtools/lib/src/memory/memory_service.dart

```diff
@@ -32,7 +32,7 @@ Future<InstanceRef> evaluate(String objectRef, String expression) async {
 Future<InboundReferences> getInboundReferences(
     String objectRef, int maxInstances) async {
   // TODO(terry): Expose as a stream to reduce stall when querying for 1000s of
-  // instances.
+  // TODO(terry): instances.
```

Comment:
nit: remove the second TODO(terry) and optionally move the word instances to the previous line.

Reply:
// TODO(terry): Expose a stream to reduce stalls querying 1000s of instances.